src/client/components/modals/Organization.js

```diff
@@ -0,0 +1,244 @@
+import React, { useContext, useState } from 'react'
+import { Field, Text, Button, theme } from '@aragon/ui'
+import PropTypes from 'prop-types'
+import { ModalWrapper, DisplayErrors } from './ModalWrapper'
+import { validateDAOAddress } from '../../utils/validation'
+import { BoxContext } from '../../wrappers/box'
+import { ModalContext } from '../../wrappers/modal'
+import { close } from '../../stateManagers/modal'
+import { isEmpty } from 'lodash'
+import styled from 'styled-components'
+import {
+  requestedCheckMembership,
+  requestedCheckMembershipClean,
+  requestedCheckMembershipSuccess,
+  requestedCheckMembershipError,
+} from '../../stateManagers/box'
+import {
+  TextInputWithValidation,
+  AnimationLoadingCircle,
+  AddOrganizationSuccess,
+  AddOrganizationError,
+} from '../../components/styled-components'
+
+const Organization = ({
```

Comment:
Can we rename to `OrganizationModal`?

Reply:
Sure, but no other modal component has "modal" in its name, should we make an exception for this one?